lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -370,13 +371,15 @@
       - Rolling update parallelism.
       - Corresponds to the C(--update-parallelism) option of C(docker service create).
   update_failure_action:
-    type: int
+    type: str
```

Comment:
Thanks for spotting and fixing this one!

Reply:
Please note that I also fixed this in #52402, which just got merged; the conflict you currently have is probably related to this line. I think git should have no problem rebasing to current `devel` (since it will notice that this PR does the same as is already done on `devel`), so resolving it (via rebase) should be no problem.